cmake/onnxruntime_python.cmake

```diff
@@ -224,6 +224,7 @@ if (onnxruntime_BUILD_UNIT_TESTS)
   file(GLOB onnxruntime_python_test_srcs CONFIGURE_DEPENDS
       "${ONNXRUNTIME_ROOT}/test/python/*.py"
       "${ORTTRAINING_SOURCE_DIR}/test/python/*.py"
+      "${ORTTRAINING_SOURCE_DIR}/test/python/*.json"
```

Comment:
first time seeing including *.json in the build list... 
I suspect this can be removed. Also wondering why we need this?

Reply:
ORTModule + Deepspeed end-to-end test uses a json file as configuration

Reply:
This will be moved to pytorch/ort repo when the time comes